app/routes/manage/index.js

```diff
@@ -93,6 +91,9 @@ export default AuthenticateRoute.extend({
             limit: "0"
           }
         }
+      }).catch((err) => {
+        console.log('Failed to get folder', err);
+        return;
```

Comment:
Very minor point, but because this happens in a Promise callback/error-handler, I don't believe that this `return` statement is necessary.